src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix4x4.Impl.cs

```diff
@@ -451,9 +452,47 @@ public static Impl CreateLookAt(in Vector3 cameraPosition, in Vector3 cameraTarg
                     0
                 );
                 result.W = new Vector4(
-                    -Vector3.Dot(axisX, cameraPosition),
-                    -Vector3.Dot(axisY, cameraPosition),
-                    -Vector3.Dot(axisZ, cameraPosition),
+                    Vector3.Dot(axisX, negativeCameraPosition),
+                    Vector3.Dot(axisY, negativeCameraPosition),
+                    Vector3.Dot(axisZ, negativeCameraPosition),
+                    1
+                );
+
+                return result;
+            }
+
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            public static Impl CreateLookToLeftHanded(in Vector3 cameraPosition, in Vector3 cameraDirection, in Vector3 cameraUpVector)
```

Comment:
why can't this just pass negative cameraDirection to the one above?

Reply:
(similar question to other methods)

Reply:
This one in particular could, but that adds a hidden additional cost that the JIT can't currently optimize away. For the others the algorithms are not simply negating a single input.

Reply:
is there an issue against JIT to improve this? is this cost high enough to justify code duplication?

Reply:
> is there an issue against JIT to improve this

There are many issues tracking the broader issues that impact this space. It's not really limited to just one thing.

> is this cost high enough to justify code duplication?

This is a known perf oriented type and the amount of code duplication is not significant. We have (ignoring the formatting done to help improve readability, around 8-11 lines of pretty simple code here.